docs/zdgbook/GettingStarted.rst

```diff
@@ -2,142 +2,141 @@
-distributed as `Python eggs
-<http://peak.telecommunity.com/DevCenter/setuptools>`_.
+This chapter covers the installation of Zope and getting started with
+the development of a simple application.  This guide uses a build
+system called `Buildout <http://www.buildout.org>`_ to build the
+application.
+
+Prerequisites
+=============
+
+Make sure you have Python installed. Version 3.6 or higher is
+recommended.
+
+Creating a `VirtualEnv <https://pypi.org/project/virtualenv/>`_ is
+recommended.
+
+In order to use buildout, you have to install the ``zc.buildout``
+package.
 
+::
+
+  $ pip install zc.buildout
 
-Directory Structure
+Directory structure
 ===================
 
-To begin the application development, create a directory structure to
-place Python packages and build related files.
+To begin application development, create a directory structure for
+the Python packages and build related files.
 
 ::
 
   $ mkdir poll
   $ mkdir poll/poll_build
   $ mkdir poll/poll.main
 
-All build related files can be added inside `poll_build` directory.
-The main Python package can be added inside `poll.main` directory.
-We can make the ``poll``, a namespace package using the functionality
-provided by `pkg_resources` module included in setuptools.
+All build related files will be added inside the ``poll_build``
+directory, whereas the main Python package goes into the
+``poll.main`` directory.
 
-Bootstrapping the Build
-=======================
+Installing Zope using zc.buildout
+=================================
 
-You should have Python 2.6 or 2.7 installed in your system.  To start
-the build process, download and run `bootstrap.py`.  The
-`bootstrap.py` will download and install `setuptools` and
-`zc.buildout` packages.  Also it will create the directory structure
-and `buildout` script inside `bin` directory.
+Zope is distributed in egg format.  To install Zope
+and create an instance, create a buildout configuration file
+(``poll/poll_build/buildout.cfg``) with following content.
 
 ::
 
-  $ cd poll/poll_build
-  $ touch buildout.cfg
-  $ wget -c http://svn.zope.org/repos/main/zc.buildout/trunk/bootstrap/bootstrap.py
-  $ python2.6 bootstrap.py
+  [buildout]
+  extends = https://zopefoundation.github.io/Zope/releases/master/versions-prod.cfg
+  parts =
+      zope4
+
+  [zope4]
+  recipe = zc.recipe.egg
+  eggs =
+      Zope
 
-Installing Zope 2
-=================
+The ``[zope4]`` part uses ``zc.recipe.egg`` which will download
+``Zope`` and all its dependencies.  It will create few console
+scripts inside the ``bin`` directory.
 
-Zope 2 is distributed in egg format.  To install Zope 2 egg
-and create an instance, update buildout configuration file
-(``buildout.cfg``) with appropriate parts and recipes.
+After updating the buildout configuration, you can run the `buildout`
+command to build the system.
 
 ::
 
-  [buildout]
-  parts = zope2
-          instance
-  extends = http://download.zope.org/Zope2/index/2.13.20/versions.cfg
+  $ buildout
```

Comment:
IMHO this should be `bin/buildout`.

Reply:
With the recommendation to activate the virtualenv, buildout is added to the path and so it is fine to just use "buildout" without the bin or possibly venv/bin prefix.